### DISCUSSION TITLE
Undo button

### DISCUSSION THREAD

**Discussion by merydian** [opengisch/QField#3164] [Q&A]
We are trying to create a workflow to record biotopes. Our main problem is that when something was wrongly split or drawn etc., it ist time consuming to undo said changes. Therefore some sort of **Undo button** would be very helpful. 

Is the integration of an undo button possible? Is it maybe already planned for future versions? 

Many thanks in advance!

**Comment by m-kuhn** [comment]
This has been implemented in https://github.com/opengisch/QField/pull/4849 and is available since https://github.com/opengisch/QField/releases/tag/v3.2.0
